docs/patterns/card/patterns/explicit-themes.html

```diff
@@ -0,0 +1,21 @@
+<uxdot-pattern color-palette="light">
+  <h3 slot="heading">Lighter themed card</h3>
+  <rh-card color-palette="lighter">
+    <h2 slot="header">Card title</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla vitae elit 
+      libero, a pharetra augue. Nullam id dolor id nibh ultricies vehicula ut id 
+      elit. Donec id elit non mi porta gravida at eget metus.</p>
+    <rh-cta slot="footer"><a href="#">Call to action</a></rh-cta>
```

Comment:
```suggestion
    <rh-cta slot="footer" href="#">Call to action</rh-cta>
```

Etc

Reply:
Ok, I updated all of the `<rh-cta>` to match the updated API as suggested. Since this is a docs update, I don't believe this will be 100% accurate/reflective of the current API until Charmander goes out, eh?